debugger/src/main.rs

```diff
@@ -337,3 +366,23 @@ fn main() -> rustyline::Result<()> {
     }
     Ok(())
 }
+
+fn check_for_updates(client: Client) -> Option<String> {
+    let response = client
+        .get("https://crates.io/api/v1/crates/pest_debugger")
+        .send();
+
+    if let Ok(response) = response {
+        response.json::<serde_json::Value>().ok().and_then(|json| {
+            let version = json["crate"]["max_version"].as_str()?;
+
+            if version != VERSION {
```

Comment:
this logic may perhaps not work in some edge cases -- e.g. if I build a local "newer" version or if the upstream version was yanked or something like that... maybe a more appropriate would be to compare semvers with `>`, but that would require defining and parsing version numbers which may be an overkill